### ISSUE TITLE
Allow disabling overtype mode

### ISSUE THREAD

**Issue by icorbrey** [microsoft/vscode#236659]
I really hate overtype modes and have loved that VSCode doesn't have this included until now. I recognize that overtype modes are a popular feature and it's great that VSCode includes it now, but those of us who can't stand our editors randomly overwriting characters when we fat finger the Insert key would appreciate a way to turn this new functionality off.

Existing functionality like overtyping existing quotes or parens or what have you is fine, I just want to disable the new mode.

Thanks!

**Comment by ArturoDent** [comment]
Does removing the keybinding in your Keyboard Shortcuts for the `Toggle Overtype` command fix it for you.  You should end up with this in your `keybindings.json`:

```jsonc
{
  "key": "insert",
  "command": "-editor.action.toggleOvertypeInsertMode"
}
```

**Comment by icorbrey** [comment]
Yes, that does seem to help, thank you! Would be nice if there was a way to do this in the normal settings though

**Comment by aiday-mar** [comment]
The solution offered by @ArturoDent fixes the issue in this thread, so I will close this issue. There is no plan to permanently turn off overtype mode with a setting.

**Comment by rkeithhill-keysight** [comment]
> I really hate overtype modes and have loved that VSCode doesn't have this included until now.

This 1000%!!  Thanks for the workaround, but I wish it were an editor setting.
